Source/Orts.Simulation/Simulation/Activity.cs

```diff
@@ -1438,7 +1435,19 @@ static bool includesWagons(Train train, List<string> wagonIdList)
         /// <returns>True if all listed wagons are not part of the given train.</returns>
         static bool excludesWagons(Train train, List<string> wagonIdList)
         {
-            bool lNotFound = false;
+            bool lNotFound = true;
+            // The Cars list is a global list that includes STATIC cars.  We need to make sure that the active train/car is processed only.
+            for (int i = 0; i < train.Cars.Count; i++)
+            {
+                var car = train.Cars[i];
+                if (car.Train.TrainType == Train.TRAINTYPE.STATIC)
+                {
+                    lNotFound = true;
+                    return lNotFound;
+                }
+
+            }
+
```

Comment:
Correct me if I'm wrong, but couldn't this simplify to:
```suggestion
            // The Cars list is a global list that includes STATIC cars.  We need to make sure that the active train/car is processed only.
            if (train.TrainType == Train.TRAINTYPE.STATIC)
                return true;

            bool lNotFound = false;
```